### DISCUSSION TITLE
Generate random content for our collection for testing

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17157] [Feature Requests]
Would be nice if we could generate lots of entries for our collection with random content. Similar to https://github.com/FakerPHP/Faker which Laravel uses and I loved working with. It is great for testing.

**Comment by paescuj** [comment]
This sounds like a great idea for an extension! 👍

**Reply by SharadKumar** [reply]
Allow setting up OpenAI key and use it for content generation based on `faker` instructions. This shall allow for domain-specific content and more realistic testing.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
